cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ValidatorTransform.java

```diff
@@ -218,5 +235,9 @@ private void init(List<Validator> validators) throws ScriptException {
     String validators;
     @Description(SCRIPT_DESCRIPTION)
     String validationScript;
+
+    @Description("Lookup tables to use during transform")
```

Comment:
since we only support keyvalue table for lookups right now, will be good to include that information in the description as well right? what do you think?